airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -1233,6 +1252,22 @@ async def test_all_supported_file_types_present(self, certified_file_based_conne
             "or add them to the `file_types -> unsupported_types` list in config."
         )
 
+    @staticmethod
+    def _validate_stream_statuses(configured_catalog: ConfiguredAirbyteCatalog, statuses: List[AirbyteStreamStatusTraceMessage]):
+        """Validate all statuses for all streams in the catalogs were emitted in correct order"""
+        stream_statuses = defaultdict(list)
+        for status in statuses:
+            stream_statuses[status.stream_descriptor.name].append(status.status)
```

Comment:
A stream descriptor is the combination of name and namespace. Could there be two streams with the same name in different namespace? How would this algorithm works with this?

Reply:
What enforces that? What prevents us from having two streams with the same name but different namespace?

Reply:
oh, I get your point. 
fixed.